README.md

```diff
@@ -43,12 +42,20 @@ On Windows, you can do this in Docker Desktop>Settings>Build engine.
 On Mac, you can do this in Docker Desktop>Preferences>Docker Engine. Edit the displayed JSON so that `"buildkit": false`, then restart Docker Desktop.
 ## Test
 Run `script/test`, or follow these steps:
-1. Run `script/bootstrap` script
-2. Install dev requirements using `pip install -r requirements-dev.txt`
+1. Run `npm install`
```

Comment:
Do people not have to install node.js first?

Reply:
Indeed! I will add the procedure description.